src/components/Comment.svelte

```diff
@@ -37,7 +37,7 @@
   }
   .card-body {
     font-size: var(--font-size-small);
-    padding: 0 1rem 1rem 1rem;
+    padding: 0 1rem 0 1rem;
```

Comment:
Notice ^

Looked around and nothing seems to be broken by it.

If we don't set it to zero we get an ugly gap:

![Screenshot from 2023-05-28 15-10-12](https://github.com/radicle-dev/radicle-interface/assets/14107758/a51eaa45-6ec8-4b5d-be7c-54e076fba60d)
![Screenshot from 2023-05-28 15-09-56](https://github.com/radicle-dev/radicle-interface/assets/14107758/ad356d50-8c3e-4576-b17e-0591cfba8c1f)